main.go

```diff
@@ -81,6 +81,12 @@ func Run(client *cmd.Client, args ...string) {
 			Usage:   "Import a key file to use with the node",
 			Action:  client.ImportKey,
 		},
+		{
+			Name:    "adapter",
+			Aliases: []string{"a"},
+			Usage:   "Add a new bridge to an external adapter",
+			Action:  client.AddAdapter,
```

Comment:
Ok, had to think about this a bit, but I think`Bridge` is the way to go. I know that the story originally asked for `adapter`, so that's my fault, but Dimitri brings up a good point that "adapter" holds a lot of meaning(external vs internal).

From a ubiquitous language perspective, the Bridge is really what's being defined here. The External Adapter is really the server that is running alongside Chainlink. The Bridge facilitates communication with the External Adapter. From a Consumer's perspective, it know nothing about Adapters being core or external, it just specifies the Tasks it wants run.